homeassistant/components/airthings_ble/sensor.py

```diff
@@ -0,0 +1,183 @@
+"""Support for airthings ble sensors."""
+from __future__ import annotations
+
+import logging
+
+from airthings_ble import AirthingsDevice
+
+from homeassistant import config_entries
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import (
+    CONCENTRATION_PARTS_PER_BILLION,
+    CONCENTRATION_PARTS_PER_MILLION,
+    LIGHT_LUX,
+    PERCENTAGE,
+    PRESSURE_MBAR,
+    TEMP_CELSIUS,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import CONNECTION_BLUETOOTH
+from homeassistant.helpers.entity import DeviceInfo, EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+from .const import DOMAIN, VOLUME_BECQUEREL, VOLUME_PICOCURIE
+
+_LOGGER = logging.getLogger(__name__)
+
+SENSORS_MAPPING_TEMPLATE: dict[str, SensorEntityDescription] = {
+    "radon_1day_avg": SensorEntityDescription(
+        key="radon_1day_avg",
+        native_unit_of_measurement=VOLUME_BECQUEREL,
+        name="Radon 1-day average",
```

Comment:
Should some of these get `state_class`es?

Reply:
Ah is that good to have if there's no device class?

Reply:
State classes let HA know if it's a good field to do stats on (like averages, mins, maxes, etc). 

https://developers.home-assistant.io/docs/core/entity/sensor/#available-state-classes